### DISCUSSION TITLE
How do I enable bare slash regex?

### DISCUSSION THREAD

**Discussion by tinder-tannerbennett** [swiftlang/vscode-swift#1257] [Q&A] [answered]
I tried passing `-enable-bare-slash-regex` to the Swift build flags, but I still get errors about regexes

Edit: I'm a dummy, we're on Xcode 15.3, which has Swift 5.1, which doesn't have this feature yet

**Comment by plemarquand** [comment, marked as answer]
Can you post the result of running `swift --version`?

**Reply by tinder-tannerbennett** [reply]
Oh… 😶‍🌫️ Apologies, never mind. I didn't realize we were on such an old version of Xcode still (well, that bare slash regex was added so recently compared to the version of Xcode we're on)
